animations/pubspec.lock

```diff
@@ -157,4 +157,4 @@ packages:
     source: hosted
     version: "2.0.8"
 sdks:
-  dart: ">=2.9.0-14.0.dev <3.0.0"
+  dart: ">=2.9.0-14.0.dev <3.0.0"
```

Comment:
There should be a new line at the end

Reply:
Unusual that this file would be edited and that would be the only change. Was this done by hand somehow?

Given the nature of the fix, @mohak852, you can probably just revert this file entirely.